tests/foreman/cli/test_usergroup.py

```diff
@@ -196,20 +197,24 @@ def test_positive_automate_bz1426957(ldap_auth_source, function_user_group):
     """
     ext_user_group = make_usergroup_external(
         {
-            'auth-source-id': ldap_auth_source['id'],
+            'auth-source-id': ldap_auth_source[1].id,
```

Comment:
What does the list indice 1 returns, is it always static ?

Reply:
`ldap_auth_source` fixture yields two objects `ldap_data` and `auth_source`  https://github.com/SatelliteQE/robottelo/blob/59b5d9dc046625703f990f65bd28fddd89185cb8/pytest_fixtures/component/satellite_auth.py#L226
Here Index 1 is authsource, so It will be always static